lib/modules/manager/npm/update/dependency/index.ts

```diff
@@ -113,10 +121,13 @@ function replaceAsString(
   throw new Error();
 }
 
-export function updateDependency({
-  fileContent,
-  upgrade,
-}: UpdateDependencyConfig): string | null {
+export function updateDependency(
+  { fileContent, upgrade }: UpdateDependencyConfig,
+  fileFormat?: string,
+): string | null {
+  const isYaml = fileFormat
+    ? fileFormat.endsWith('.yaml') || fileFormat.endsWith('.yml')
```

Comment:
What is "fileFormat"? is it the file _name_?

Reply:
Right now I don't remember, I think you are right, It was really long time ago, and this ended evolving from somewere else.